AIO-Tools/AIO-Tools.csproj

```diff
@@ -75,7 +76,7 @@
       <HintPath>..\packages\EntityFramework.6.4.4\lib\net45\EntityFramework.SqlServer.dll</HintPath>
     </Reference>
     <Reference Include="INIFileParser, Version=2.5.2.0, Culture=neutral, PublicKeyToken=79af7b307b65cf3c, processorArchitecture=MSIL">
-      <HintPath>..\packages\ini-parser.2.5.2\lib\net20\INIFileParser.dll</HintPath>
+      <HintPath>DLL\INIFileParser.dll</HintPath>
```

Comment:
Don't think this needed if you already restored the package.

Reply:
I think it did that automatically, can be ignored.

Reply:
Either you edit this back before I publish or you upload this file to DLL folder (The first might be better)